antergos/etcher/PKGBUILD

```diff
@@ -2,21 +2,23 @@
 # Original Contributor: Andrew Stubbs <andrew.stubbs@gmail.com>
 
 pkgname=etcher
-pkgver=1.1.2
+_realver=1.2.0
```

Comment:
This extra variable appears to be pointless. Could you just bump the version and update the checksums and leave out the other changes? Thanks!